next.config.js

```diff
@@ -1,38 +1,19 @@
 import { join, dirname } from "path";
 import { fileURLToPath } from "url";
+import { createJiti } from "jiti";
 
 const __filename = fileURLToPath(import.meta.url);
 const __dirname = dirname(__filename);
 
-let API_URL;
```

Comment:
`jiti` allows TypeScript files to be imported into next.config, could be removed after updating to Next 15.

This import will type-check the environment at build time. Any invalid env will error the build process immediately.